digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -4,20 +4,24 @@ import fi.liikennevirasto.digiroad2.linearasset.PolyLine
 
 object GeometryUtils {
 
-  // Default value of minimum distance where locations are considered to be same
+  /** Default value of minimum distance where locations are considered to be same */
   final private val DefaultEpsilon = 0.01
+  /** If links are more than <i>MaxDistanceForConnectedLinks</i> apart, they are considered to being disconnected. */
   val MaxDistanceForConnectedLinks = 0.1
 
+  /** @return the first, and last the point of the given <i>geometry</i> as a tuple. */
```

Comment:
ylimääräinen "the"

Reply:
tai väärä sanajärjestys

Reply:
Korjattu.
(Tou, sarjatuotteena lisäillyt artikkelit osuvat näemmä mihin sattuu)